contrib/storage-jdbc/src/main/java/org/apache/drill/exec/store/jdbc/writers/JdbcBigintWriter.java

```diff
@@ -31,9 +31,9 @@ public JdbcBigintWriter(String colName, RowSetLoader rowWriter, int columnIndex)
 
   @Override
   public void load(ResultSet results) throws SQLException {
-    boolean b = results.wasNull();
-    if (! results.wasNull()) {
-      long value = results.getLong(columnIndex);
+    // clickhouse requires getting the column before checking nullability
+    long value = results.getLong(columnIndex);
+    if (!results.wasNull()) {
```

Comment:
Good catch! As it turns out, the original code is wrong for all DBs, not just Clickhouse. See [this article](https://schneide.blog/2021/02/22/jdbcs-wasnull-method-pitfall/).

Suggestion, change the comment to something like "JDBC reports nullability only after getting the column value." Here and below.